### DISCUSSION TITLE
lotus-bee全方位代码优化

### DISCUSSION THREAD

**Discussion by beelant** [filecoin-project/community-china#111] [软件优化]
https://github.com/beelant/lotus-bee
蜂灯科技凭借分布式存储领域开发经验和对Lotus代码的深刻钻研，特此开发Lotus-Bee版本Lotus代码，提供广大分布式存储提供商更快捷的数据封装速度，更灵活的集群配置方式，更稳定的运维特性的Lotus代码，为推动行业发展贡献绵薄之力，现决定开放免费使用。

**Comment by ncgege** [comment]
感谢
